src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
 
+        Machine[] machines = {truck, bulldozer, excavator};
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
The messages indicating the type of machine working should be more informative. It should be clear from the output which specific type of Machine ('Truck', 'Bulldozer', or 'Excavator') is performing the work or stopping. Ensure that the doWork() and stopWork() methods in each specific Machine subclass print out informative messages.